x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/EsqlExpressionTranslators.java

```diff
@@ -65,25 +93,27 @@
 import static org.elasticsearch.xpack.ql.util.NumericUtils.unsignedLongAsNumber;
 
 public final class EsqlExpressionTranslators {
+    private static final String DATE_FORMAT = "strict_date_optional_time_nanos";
+    private static final String TIME_FORMAT = "strict_hour_minute_second_fraction";
 
     public static final List<ExpressionTranslator<?>> QUERY_TRANSLATORS = List.of(
         new EqualsIgnoreCaseTranslator(),
         new BinaryComparisons(),
         new SpatialRelatesTranslator(),
-        new ExpressionTranslators.Ranges(),
-        new ExpressionTranslators.BinaryLogic(),
-        new ExpressionTranslators.IsNulls(),
-        new ExpressionTranslators.IsNotNulls(),
-        new ExpressionTranslators.Nots(),
-        new ExpressionTranslators.Likes(),
-        new ExpressionTranslators.InComparisons(),
-        new ExpressionTranslators.StringQueries(),
-        new ExpressionTranslators.Matches(),
-        new ExpressionTranslators.MultiMatches(),
+        new Ranges(),
+        new BinaryLogic(),
+        new IsNulls(),
+        new IsNotNulls(),
+        new Nots(),
+        new Likes(),
+        new InComparisons(),
+        new StringQueries(),
+        new Matches(),
+        new MultiMatches(),
```

Comment:
The corresponding tests still need to be migrated before this is ready for review.